sowingdev/app/curriculum/CodeBlock.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { CopyBlock, dracula } from 'react-code-blocks';
+
+
+const CodeBlock: React.FC<{code:any}> = ({code}: {code: any}) => {
```

Comment:
Can we update this to include typing? `code:any`, code should be used with Typescript, we should never use `any` unless there is a good reason for it.